### PR TITLE
Add some CODEOWNERS to itsmeow

### PR DESCRIPTION
**Pull request by itsmeow**
## About The Pull Request

Adds the following:

- TGUI core (not interfaces, which is what most use)
- Map tile aggregator
- Player panel

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Wait why the ticket system though?

**Review by ike709 — APPROVED** [review]

File: .github/CODEOWNERS
Comment (Crossedfall) [inline .github/CODEOWNERS]:
```suggestion
```
I don't think you need to own fugitives. It's a ported game mode.

File: .github/CODEOWNERS
Comment (Crossedfall) [inline .github/CODEOWNERS:34]: I'm not sure who actually gets code owner here. If it would give it to you for being a specific file or me for owning the directory and being higher in the list. Probably fine?

File: .github/CODEOWNERS
Comment (Crossedfall) [inline .github/CODEOWNERS]: Eeehhh. You did do the most recent big changes, but, I don't know if there is any reason to have you specifically own these files.

**Review by Crossedfall — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (itsmeow) [reply .github/CODEOWNERS]: To receive review requests on them.

File: .github/CODEOWNERS
Comment (itsmeow) [reply .github/CODEOWNERS:34]: Not sure

File: .github/CODEOWNERS
Comment (itsmeow) [reply .github/CODEOWNERS]: I also rewrote almost all the fugitives code, not as part of a port

Comment (PowerfulBacon) [reply .github/CODEOWNERS]: Personally I don't think its complex / enough of a framework system to deem it super necessary, you can't really mess up edits to fugative code and everyone can review it equally as well.

File: .github/CODEOWNERS
Comment (itsmeow) [reply .github/CODEOWNERS:34]: I think it would consider us both as owning it

File: .github/CODEOWNERS
Comment (Crossedfall) [reply .github/CODEOWNERS]: @PowerfulBacon thoughts?

Comment (PowerfulBacon) [reply .github/CODEOWNERS]: Personally I don't see any huge reason to have it specifically assigned to one person (assuming we respect the code ownership) due to the amount of work a lot of different people have put into this system and it still being mostly similar to the original system that we have all worked with before. But at the same time I also don't particularly see any reason against it since we can just ignore code owner requests, but then that kind of defeats the point of it since then you might as well just watch the PR list and check which ones modify an admin file.
For more contributed files like this (where the bulk of it has been made by a large number of different people), I'm not too sure since if we just give code ownership anytime someone does some changes to it then we'd have the entire codebase on code review when every maintainer is equally good at reviewing it, which defeats the purpose of having code owners in the first place since now we are just ignoring them entirely. (For example, I originally had the shuttles subsystem before it was made specific to supercruise, since shuttles was such a broad area and loads of different people made that, so it was just an inconvenience for me and other people)

**Review by PowerfulBacon — APPROVED** [review]

File: .github/CODEOWNERS
Comment (Crossedfall) [inline .github/CODEOWNERS:35]:
```suggestion
```

Comment (Crossedfall) [reply .github/CODEOWNERS:35]: Same sort of deal

File: .github/CODEOWNERS
Comment (Crossedfall) [reply .github/CODEOWNERS]: Yeah, I view code ownership as either being, "This person is the defacto subject-matter expert and really any changes should go through them because it would take more effort for anyone else to review." (e.g. Kat with TGUI components) OR if someone specific should approve of changes before they get deployed in some way (e.g. me and configs)

Stuff that is fairly general and doesn't require explicit approval shouldn't have code ownership.

File: .github/CODEOWNERS
Comment (itsmeow) [reply .github/CODEOWNERS:35]: I wrote this ENTIRE file and know everything about how it works

Comment (Crossedfall) [reply .github/CODEOWNERS:35]: Discussed in Discord. We can leave this one I suppose.

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**
